From a5bb3d9b872954cf1ef0bcdd90dc208b3debd736 Mon Sep 17 00:00:00 2001
From: Robin Dunn <robin@alldunn.com>
Date: Wed, 7 Jun 2023 19:27:55 -0700
Subject: [PATCH] Bump version number

---
 CHANGES.rst           | 14 ++++++++++++++
 buildtools/version.py |  4 ++--
 2 files changed, 16 insertions(+), 2 deletions(-)

diff --git a/CHANGES.rst b/CHANGES.rst
index 250c5c8b2a11..8abf72546140 100644
--- a/CHANGES.rst
+++ b/CHANGES.rst
@@ -8,6 +8,20 @@
 wxPython Changelog
 ==================
 
+
+4.2.2
+------
+* (unreleased)
+
+PyPI:   https://pypi.python.org/pypi/wxPython/4.2.2
+Extras: https://extras.wxPython.org/wxPython4/extras/
+Pip:    ``pip install wxPython==4.2.2``
+
+New and improved in this release:
+
+
+
+
 4.2.1 "Size matters not." (Yoda)
 --------------------------------
 * 7-June-2023
diff --git a/buildtools/version.py b/buildtools/version.py
index aa3c26593148..31a85afe37d7 100644
--- a/buildtools/version.py
+++ b/buildtools/version.py
@@ -24,9 +24,9 @@ PROJECT_NAME     = 'wxPython'
 # version numbers.
 VER_MAJOR        = 4
 VER_MINOR        = 2
-VER_RELEASE      = 1
+VER_RELEASE      = 2
 
-VER_FLAGS        = ""     # wxPython release flags
+VER_FLAGS        = "a1"     # wxPython release flags
 
 # The VER_FLAGS value is appended to the version number constructed from the
 # first 3 components and should be set according to the following patterns.
-- 
2.44.0

